src/promptflow/setup.py

```diff
@@ -45,7 +45,7 @@
     "waitress>=2.1.2,<3.0.0",  # used to serve local service
     "opencensus-ext-azure<2.0.0",  # configure opencensus to send telemetry to azure monitor
     "ruamel.yaml>=0.17.10,<1.0.0",  # used to generate connection templates with preserved comments
-    "pyarrow>=14.0.1,<15.0.0",  # used to read parquet file with pandas.read_parquet
+    "fastparquet>=2024.2.0",  # used to read parquet file with pandas.read_parquet
```

Comment:
Do we have any document on this package? 
Does it have any limiation compare to pyarrow?

Reply:
https://fastparquet.readthedocs.io/en/latest/index.html

The known limitation is that `fastparaquet` can be slower when we try to read a huge dataset.
Advantages:
- size of installed environment is smaller by ~200MB
- size of downloaded whl smaller by ~20MB